tests/test_client.py

```diff
@@ -214,8 +213,8 @@
                 "com.facebook.presto.sql.tree.Table.accept(Table.java:50)",
                 "com.facebook.presto.sql.tree.AstVisitor.process(AstVisitor.java:22)",
                 "com.facebook.presto.sql.analyzer.StatementAnalyzer.analyzeFrom(StatementAnalyzer.java:1413)",
-                "com.facebook.presto.sql.analyzer.StatementAnalyzer.visitQuerySpecification(StatementAnalyzer.java:670)",
-                "com.facebook.presto.sql.analyzer.StatementAnalyzer.visitQuerySpecification(StatementAnalyzer.java:166)",
+                "com.facebook.presto.sql.analyzer.StatementAnalyzer.visitQuerySpecification(StatementAnalyzer.java:670)",  # NOQA
```

Comment:
Is it possible to do this more cleanly? For the entire block at once?

Reply:
Not to my knowledge. You can make this cleaner on line continuations, but not for arrays. Another option would be for the whole file, but I don't think that is better at this point. 

I am going to update this to only remove checks for line length though.